cmd/environment.go

```diff
@@ -21,17 +22,30 @@ func (h *Handler) Environment(ctx context.Context, req *entity.CommandRequest) e
 
 	var environment *entity.Environment
 	if len(req.Args) > 0 {
-		// Create new environment
-		environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
-			Name:      req.Args[0],
-			ProjectID: project.Id,
-		})
-		if err != nil {
-			return err
+		var name = req.Args[0]
+
+		// Look for existing environment with name
+		for _, projectEnvironment := range project.Environments {
+			if name == projectEnvironment.Name {
+				environment = projectEnvironment
+			}
+		}
+
+		if (environment != nil) {
+			fmt.Printf("%s Environment: %s\n", promptui.IconGood, ui.BlueText(environment.Name))
+		} else {
+			// Create new environment
+			environment, err = h.ctrl.CreateEnvironment(ctx, &entity.CreateEnvironmentRequest{
+				Name:      name,
+				ProjectID: project.Id,
+			})
+			if err != nil {
+				return err
+			}
+			fmt.Println("Created Environment ✅\nEnvironment: ", ui.BlueText(ui.Bold(name).String()))
```

Comment:
Should this check emoji also be the prompui one?

Reply:
Probably, but may not work on terminals that don't support emojis, but let's assume it's fine.